src/extractor/git.rs

```diff
@@ -80,6 +80,10 @@ pub struct Object {
 pub struct Git {
     pub metrics: Metrics,
     pub objects: HashMap<String, Object>,
+    //the ref_target is the ref used to traverse git the
```

Comment:
I think something is wrong with this comment, maybe you meant "the git repository tree" instead of "git the repository tree" ?